npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
@ccgamble I changed the file structure and I believe with the latest commit this issue is fixed.